bundles/org.openhab.binding.elroconnects/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,268 @@
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="heatalarm">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Heat Alarm</label>
+		<description>ELRO Connects heat alarm</description>
+		<channels>
+			<channel id="heatalarm" typeId="heatalarm"/>
+			<channel id="mutealarm" typeId="mutealarm"/>
+			<channel id="testalarm" typeId="testalarm"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="wateralarm">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Water Alarm</label>
+		<description>ELRO Connects water alarm</description>
+		<channels>
+			<channel id="wateralarm" typeId="wateralarm"/>
+			<channel id="mutealarm" typeId="mutealarm"/>
+			<channel id="testalarm" typeId="testalarm"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="entrysensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Door/Window Sensor</label>
+		<description>ELRO Connects door/window contact sensor</description>
+		<channels>
+			<channel id="entryalarm" typeId="entryalarm"/>
+			<channel id="entry" typeId="entry"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="motionsensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Motion sensor</label>
```

Comment:
```suggestion
		<label>Motion Sensor</label>
```
Labels should have all words start with a capital. Please change all that need this done.